### PR TITLE
Package.json dependencies updated

### PR DESCRIPTION
**Pull request by lguzzon**
Package.json dependencies updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,54 +44,54 @@
...
+    "inquirer": "^1.2.3",
     "jsonschema": "^1.0.2",
-    "jsonwebtoken": "^5.4.0",
+    "jsonwebtoken": "^7.3.0",
     "lodash": "^4.6.1",
     "open": "^0.0.5",
-    "ora": "0.2.3",
-    "portfinder": "^0.4.0",
-    "progress": "^1.1.8",
+    "ora": "1.2.0",
```
Comment (rockwotj) [inline package.json]: We can't upgrade `ora` because after `0.2.3` dropped support for older versions of node that we support.

**Comment by lguzzon** [comment]
Waiting patiently ...

**Comment by mbleigh** [comment]
Please remove the changes to the non-main `package.json` file as they are tracked separately and not part of the public dependency graph anyway.

**Comment by bkendall** [comment]
This is now pretty outdated, and I don't see it being merged any longer without a lot of serious work to get it up to date and tested. Thank you for the effort though, it really is appreciated 😄

**Pull request closed without merging**